docs/elastic_inference.md

```diff
@@ -0,0 +1,182 @@
+# Model Serving with Amazon Elastic Inference 
+
+## Contents of this Document
+* [Introduction](#introduction)
+* [Custom Service](#custom-service)
+* [Creating a EC2 instance with EIA support](#creating-a-ec2-instance-with-eia-support)
+* [Custom Service file with EIA](#custom-service-file-with-eia)
+* [Running elastic inference on a resnet-152](#running-elastic-inference-on-a-resnet-152)
+## Introduction
+
+Amazon Elastic Inference allows you to attach low-cost GPU-powered acceleration to Amazon EC2 and Amazon SageMaker instances to reduce the cost of running deep learning inference by up to 75%. With MMS it is easy to deploy an MXNet based model, taking advantage of the attacble hardware accelerator.
+In this document we explore using Elastic Inference Acclerator (EIA) attached to a Compute Optimized EC2 instance.
```

Comment:
Can you add something to clarify how EI isn't EIA? In the previous paragraph you introduce the accelerator, so just carry on with that.